Content.Server/Body/Systems/BrainSystem.cs

```diff
@@ -33,12 +39,20 @@ private void HandleMind(EntityUid newEntity, EntityUid oldEntity)
             if (HasComp<BodyComponent>(newEntity))
                 ghostOnMove.MustBeDead = true;
 
+            if (HasComp<UnborgableComponent>(oldEntity)) // DeltaV
+                EnsureComp<UnborgableComponent>(newEntity);
+
             if (!_mindSystem.TryGetMind(oldEntity, out var mindId, out var mind))
                 return;
 
             _mindSystem.TransferTo(mindId, newEntity, mind: mind);
         }
 
+        private void OnExamined(Entity<UnborgableComponent> ent, ref ExaminedEvent args) //DeltaV
```

Comment:
put this in an UnborgableSystem also this makes no sense to just look at someone and know they cant be borged, return early if the entity has BodyComponent

Reply:
imo its slightly better to not have people waste time shoving brain into an MMI if it was never going to work. Maybe it could be "This brain is incompatible with being digitized due to its oversized Frontal Lobe"

Reply:
that should be on health analyzer like unreviveable, you cant just look at someone and know that